neqo-crypto/src/p11.rs

```diff
@@ -99,6 +99,7 @@ impl std::fmt::Debug for SymKey {
 }
 
 /// Generate a randomized buffer.
+#[must_use]
```

Comment:
I've landed this bit on master.  I have no idea how the other changes landed without hitting this.